odoo/tools/translate.py

```diff
@@ -456,7 +462,23 @@ def __call__(self, source):
                 cr.close()
         return res
 
+
+class LazyStrTranslator:
+
+    __slots__ = ['_source', '_']
+    def __init__(self, source):
+        self._ = GettextAlias()
```

Comment:
Can't we use `_` from the package scope? Why do we need a new instance every time?

Reply:
I guess we can reuse it, indeed